src/bp-templates/bp-nouveau/buddypress/groups/groups-loop.php

```diff
@@ -17,69 +17,106 @@
 	</div>
 <?php endif; ?>
 
+<?php
+
+	$group_cover_height = bb_get_group_cover_image_height();
+	$cover_class		= ( bp_disable_group_cover_image_uploads() || ( function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('cover-images') ) ) ? 'bb-cover-disabled' : 'bb-cover-enabled';
+	$meta_privacy		= function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('group-privacy') ? 'meta-privacy-hidden' : '';
+	$meta_group_type	= function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('group-type') ? 'meta-group-type-hidden' : '';
+	$group_members		= function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('members') ? 'group-members-hidden' : '';
+	$group_alignment	= bp_get_option('bb-group-directory-layout-grid-style', 'centered');
+
```

Comment:
```suggestion
	$group_cover_height = bb_get_group_cover_image_height();
	$cover_class		= ( bp_disable_group_cover_image_uploads() || ( function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('cover-images') ) ) ? 'bb-cover-disabled' : 'bb-cover-enabled';
	$meta_privacy		= function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('group-privacy') ? 'meta-privacy-hidden' : '';
	$meta_group_type	= function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('group-type') ? 'meta-group-type-hidden' : '';
	$group_members		= function_exists('bb_platform_group_element_enable') && !bb_platform_group_element_enable('members') ? 'group-members-hidden' : '';
	$group_alignment	= bp_get_option('bb-group-directory-layout-grid-style', 'centered');

```

Follow the spaces properly

Reply:
This is group directory related changes
CC: @krunal-buddyboss

Reply:
Done